### DISCUSSION TITLE
Trying to run on lutris for Linux getting an error.

### DISCUSSION THREAD

**Discussion by Fggrs** [krypto5863/COM-Modular-Installer#16] [General]
Started initial process 471 from gamemoderun /app/bin/wine /home/deck/Desktop/
" N DeKtOpI tch
Save/COM Modular Installer 2.5.17.1.exe
start monitoring process.
gamemodeauto:
Assertion at /vagrant/mono/mono/metadata/marshal.C:4252, condition `is ok (
error)' not met, function:mono marshal get vtfixup ftnptr, Could not load type
of field 'CMIHelper, CmiHelper:MainEngine' (0) due to: Could not load file or
assembly 'knyaz.Optimus, Version=3.0.1.8, Culture=neutral, PublickeyToken=null
or one of its dependencies. assembly:C:\users\deck\Temp\is-₩35NP.tmp\cMIHel-
per.dll type:CmiHelper member: (null)
Monitored process exited.
Initial process has exited (return code: 768)
Exit with return code 768

Any help would be appreciated.

**Comment by krypto5863** [comment]
Sorry for the late reply. You're not the first to attempt this.

I've _heard_ some guy mention that you need to manually download Kynaz.Optimus from nuget and place it somewhere CMI can access it. Possible in the same directory CMI is in, haven't really messed with it.

Nuget packages are just glorified zip files so you can download them manually from their website and extract the aforementioned dll.

You may also need the proper Net SDKs.

Mind you I don't officially support this, the best Linux user I've known recommends a VM over emulation but the choice is ultimately yours, I myself don't use Linux for this.

**Comment by krypto5863** [comment]
Should no longer be an issue.
